nix: fix gomobile builds requiring Android SDK

jakubgs opened the pull request:
This works fine on `x86_64`, but will fail when used on `aarch64`. Also upgrades Nixpkgs.

Notable package upgrades:

| Software  | Old          | New           |
|-----------|--------------|---------------|
| Glibc     | `2.32`       | `2.33`        |
| BinUtils  | `2.35.1`     | `2.35.2`      |
| CoreUtils | `8.32`       | `9.0`         |
| OpenSSL   | `1.1.1k`     | `1.1.1l`      |
| Git       | `2.31.1`     | `2.33.1`      |
| GCC       | `8.4.0`      | `8.5.0`       |
| Clojure   | `1.10.3.855` | `1.10.3.1029` |
| Node.js   | `12.22.1`    | `12.22.7`     |
| Yarn      | `1.22.10`    | `1.22.17`     |

Issue: https://github.com/status-im/status-react/issues/12794

status-im-auto commented:
### Jenkins Builds
| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_check_mark: | 31da47d8 | [#1](https://ci.status.im/job/status-react/job/prs/job/ios/job/PR-12797/1/) | 2021-11-05 13:56:04 | ~23 min | `ios` | [:package:`ipa`](https://i.diawi.com/gBL9WK) [:calling:](https://chart.apis.google.com/chart?cht=qr&chs=400x400&chld=L%7C%0A1&chl=https%3A%2F%2Fi.diawi.com%2FgBL9WK)|

Jonathansoufer reviewed (approved):
Tested locally and it fixes that Android SDK issue. LGTM!

jakubgs commented:
I tried using version with my fix merged from official `nixpkgs`: https://github.com/NixOS/nixpkgs/commit/3e09ae93

But it fails with:
```
searching for dependencies of toolchains/llvm/prebuilt/linux-x86_64/lib64/libc++.so
autoPatchelfHook could not satisfy dependency liblog.so wanted by toolchains/llvm/prebuilt/linux-x86_64/lib64/clang/11.0.5/lib/linux/libclang_rt.scudo-x86_64-android.so
Add the missing dependencies to the build inputs or set autoPatchelfIgnoreMissingDeps=true
builder for '/nix/store/xrmvjh582pkgqf3qhrh0vpa4lw5bs20l-ndk-bundle-22.1.7171670.drv' failed with exit code 1
```
Which appears to be something new...

jakubgs committed:
nix: upgrade nixpkgs to current master

Notable package upgrades:

| Software  | Old          | New           |
|-----------|--------------|---------------|
| Glibc     | `2.32`       | `2.33`        |
| BinUtils  | `2.35.1`     | `2.35.2`      |
| CoreUtils | `8.32`       | `9.0`         |
| OpenSSL   | `1.1.1k`     | `1.1.1l`      |
| Git       | `2.31.1`     | `2.33.1`      |
| GCC       | `8.4.0`      | `8.5.0`       |
| Clojure   | `1.10.3.855` | `1.10.3.1029` |
| Node.js   | `12.22.1`    | `12.22.7`     |
| Yarn      | `1.22.10`    | `1.22.17`     |

Signed-off-by: Jakub Sokołowski <jakub@status.im>

jakubgs committed:
nix: fix gomobile builds requiring Android SDK

This works fine on `x86_64`, but will fail when used on `aarch64`.

Issue: https://github.com/status-im/status-react/issues/12794

Signed-off-by: Jakub Sokołowski <jakub@status.im>

jakubgs commented:
They fixed the Android NDK build in: https://github.com/NixOS/nixpkgs/commit/15111f8a

Jonathansoufer reviewed (approved):
LGTM!

status-im-auto commented:
## 99% of end-end tests have passed
```
Total executed tests: 69
Failed tests: 1
Passed tests: 68
```
```
IDs of failed tests: 6646 
```
<h3>Failed tests (1)</h3><details><summary>Click to expand</summary><br/><li><a href="https://ci.status.im/job/end-to-end-tests/job/status-app-prs-rerun/parambuild/?BRANCH_NAME=develop&APK_NAME=12797.apk&PR_ID=12797&TR_CASE_IDS=6646">Rerun tests</a></li><br/><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2369735">test_set_profile_picture</a>, id: 6646 </b></td></tr><tr><td><p><blockquote><div>Device 1: Tap on found: Button </div><div>Device 1: Find 'Button' by 'accessibility id': `Crop` </div></blockquote></p><code> Device 1: Button by accessibility id: `Crop` is not found on the screen</code><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/7f32180090ea459eb537a0882ca84ee8?auth=9cdad7941e5d132aaf3f2111e7aa59cb">Steps, video, logs</a></li><li><a href="https://assets.saucelabs.com/jobs/7f32180090ea459eb537a0882ca84ee8/0048screenshot.png?auth=9cdad7941e5d132aaf3f2111e7aa59cb">Failure screenshot</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/4591a9dbcc2b4568a43495ad6fe6e6fa?auth=476356d6918b3d74a1e34594e940c073">Steps, video, logs</a></li><li><a href="https://assets.saucelabs.com/jobs/4591a9dbcc2b4568a43495ad6fe6e6fa/0025screenshot.png?auth=476356d6918b3d74a1e34594e940c073">Failure screenshot</a></li></ul></p></ul></p></td></tr></tbody></table></details><h3>Passed tests (68)</h3><details><summary>Click to expand</summary><br/><br/><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2369729">test_image_in_one_to_one_send_save_reply_timeline</a>, id: 6305 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/af7c79e6908344438f17b9d1b2599959?auth=b5c4544e00a48c6fe7f9707e24d42f58">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/7c67178e38234af79a58f8499c78fc89?auth=36afe8c67d1ce41719fbc26eb2aeecd1">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2369711">test_add_account_to_multiaccount_instance_generate_new</a>, id: 6224 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/721356ed8e8748569c56d818d040d081?auth=fef526cfa77b901a8aa569c1bf46b809">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2369700">test_can_add_existing_ens_on_mainnet</a>, id: 5502 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/0e5c4edbe83a4f74b9248d9e1221fbd4?auth=d588f9d66f38adb9844ad6b35eb1511c">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2369672">test_request_stt_from_dapp</a>, id: 5309 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/47ac167f7e8e4d3bb1252830e13d6c8c?auth=d5c7be0dc4db5d42e43761288ec5fb93">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2369726">test_keycard_can_see_all_transactions_in_history</a>, id: 6291 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/29565926e35349a09a82fb676b125527?auth=3fef6a4dd79114ee7c570acfbbc7753e">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2369683">test_sign_message_and_2tx_in_batch_and_transactions_filters_from_daap</a>, id: 5342 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/441ec24ff9dc4004a5c57c7dab31f112?auth=1545f5c76b3b90b4e7ffdbb5b43ef932">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>7. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2369716">test_add_and_delete_watch_only_account_to_multiaccount_instance</a>, id: 6244 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/d4b04698297841b0a58e3dda850274fd?auth=7eac404cc32a1c0de51149633ccd2eeb">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>8. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2369710">test_open_blocked_secure_not_secure_inlalid_offline_urls</a>, id: 6210 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/0a87b94ef1d2401baf1b581c3e32f53a?auth=2ea3001339eaf57c88006e211098c9cd">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>9. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2369696">test_open_public_chat_using_deep_link</a>, id: 5396 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/4785dd4c5f7441509302e307d6089392?auth=a5451c73d0965037464599750b8e409f">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>10. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2369669">test_offline_add_new_group_chat_member</a>, id: 3998 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/bff8a1a06c654ff8948b2e0a6e8d076c?auth=b0b596c229de798ec13b75ab2f5bc4af">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/7cc972708242452096059df928428cd9?auth=7f5a7b4b68859d1fa9abe580431374cf">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://saucelabs.com/jobs/0ac2cc51690f416199405bb332d35ed4?auth=e981fc5c9fd79aa8aa20c2e465143f70">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>11. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2369676">test_send_non_english_message_to_newly_added_contact_on_different_networks</a>, id: 5315 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/28373e1e984048ceb0f53c0df7f103f9?auth=907850e2325463cfbfb32f3d46d94c1c">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/74fce6264dd14316b1eb9b325592bcff?auth=b14b2b5fab4cc5e46945f83e1db4b553">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>12. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2369675">test_can_see_balance_and_all_transactions_history_on_cellular</a>, id: 5314 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/fb40f71af91142df834426bfe2e3077a?auth=a8ba50cdcbbbb3cddf62574b16954a2b">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>13. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2369698">test_insufficient_funds_wallet_positive_balance</a>, id: 5412 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/3bf4aca962d9451f8f617986401a3e79?auth=6f2c55c1493f6564647499c52d1afbb4">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>14. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2369694">test_delete_close_all_tabs</a>, id: 5390 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/8f717c0d6ac941ce975cb4ee1e65c65c?auth=cf1ca7c26d2ed94cb6ad5364f735f152">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>15. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2369668">test_create_new_group_chat_messaging_pn_delivered</a>, id: 3994 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/f7a4b8a40eda44ab91c6de731df0b393?auth=cd562c2151a2d54ee6e1e01c1a937609">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/5e058e7d643b4744abf5543509607443?auth=00b0ebda0727ec211e23ee9fd2c1057a">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>16. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2369685">test_send_token_with_7_decimals</a>, id: 5350 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/a4a66a037f35412799046bc031669a0d?auth=0d9b575acc45e0d0afcf31115c559b2c">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>17. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2369693">test_delete_chats_via_delete_button_rejoin</a>, id: 5387 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/8ccdf50edf444d0d8b5ac7df62973424?auth=949cab62abf5fd994b6187cf18f47cbf">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>18. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2369732">test_send_transaction_set_recipient_options</a>, id: 6328 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/ebee1e0e6dce4489a05c3610a1ada81b?auth=d2516657dc0ec1a353f39c67726231a5">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>19. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2369718">test_keycard_sign_message_and_transactions_from_daap</a>, id: 6251 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/17ca76d559f74a04aa3e4b1cba87cabf?auth=091e0cb0481b0b150d9d56f828640197">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>20. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2369728">test_recover_account_from_new_user_seedphrase</a>, id: 6296 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/8ede8a34e7ae4972a20ff9c8b0c6651d?auth=7acebf78a3fb4004f3c07e29c671dd24">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>21. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2369704">test_keycard_can_recover_keycard_account_card_pairing</a>, id: 5758 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/8a7d1ec445a94815bd09877adc4dc207?auth=74c3b4eb57125f46583c8d7087939dec">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>22. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2369708">test_block_user_from_public_chat</a>, id: 5786 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/6ebf59ff227347a1a205f3bc2c49fa97?auth=36c72389044909d74c50fef600125d10">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/184e023d878e43bdbe25319960d42ee4?auth=3c35f0c2ff9036f16b1d2e49e242d91c">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>23. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2369731">test_send_audio_message_with_push_notification_check</a>, id: 6316 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/bc610d3420c04d9291cc6ebc94698080?auth=0c9d1f34ffeb17a7dd35b1b5e3779a43">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/ae56e3624b12420296ddca6bf6542a1d?auth=0fff8772db9bd69efb12ae216e1a84dc">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>24. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2369701">test_redirect_to_public_chat_tapping_tag_message_fetch_more_history</a>, id: 5675 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/c91d047eeb3946e6928176d43e8d5d3c?auth=b9a583db25a92345444373bc4b9befb4">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>25. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2369734">test_restore_account_migrate_multiaccount_to_keycard</a>, id: 6645 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/0d52f8380bf642b8b0bdf50f553b2ef7?auth=62700049bc3267e1329a68a64bddb9f4">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>26. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2369736">test_edit_delete_message_in_one_to_one_and_public_chats</a>, id: 695843 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/e993055d9c304ae6b9ecbaeae126970d?auth=e6bc327a7220063a48a0d2f62bf0504d">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/bc2f12d5a50140ffae6712bf5ffa112c?auth=bd30320bdc05350c8ab9b37be9dacc6c">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>27. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2369712">test_ens_mentions_pn_and_nickname_in_public_and_1_1_chats</a>, id: 6226 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/05ccc5e9c6b2458db76984b2c95f6006?auth=e0dbd4d0d2780720d0ea570d42f7363c">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/8d40ee7379314002b21fcf54ba9ea0bb?auth=edefabc8003bed186d42636af69a89d3">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>28. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2369725">test_keycard_send_eth_from_wallet_to_address</a>, id: 6289 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/abb04e3dc9bd4c05a4254933799b55f5?auth=cef7f409a6fd6896b3642199a9c1da04">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>29. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2369702">test_keycard_create_login_restore_unlock_same_seed</a>, id: 5689 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/3d58ab9cfe9648288136ec0728147bb3?auth=3218b6882bb1943057a69c06e136b0bb">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/246f33216fc742b9b666bdbeefbda8c6?auth=dc46fa44839b66b819bfe5efa5ecf115">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>30. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2369722">test_add_account_to_wallet_private_key_and_seed_phrase</a>, id: 6272 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/17491dcebf1d49e2b8b2b349c01bd3d2?auth=b4ed1f46ebc899ba59498f1fe3743103">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>31. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2369673">test_offline_is_shown_messaging_1_1_chat_sent_delivered</a>, id: 5310 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/8a18a8c86fc64de5897080d6c1e599ef?auth=b26f7bd16a66f40e2d5c811e6af06c0c">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/5a101789b18740c6a65a22eecead0c1d?auth=93e62a79cac384019ebc7889dc4c4d41">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>32. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2369705">test_pair_devices_sync_one_to_one_contacts_nicknames_public_chat</a>, id: 5762 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/ad0c32cfe7df44cabf2f35ec85be3b20?auth=5fcb119ccbb58fba64a9645f220f4ccf">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/d554fd3cad3c4025b419fdf0942b4d1f?auth=362c7b04490a74189be75e1ea52dff9c">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://saucelabs.com/jobs/eccfdc51c2574e50a57c48e3eccee15a?auth=027ceade808ce6f027234e6cc705b68c">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>33. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2369678">test_long_press_to_delete_chat</a>, id: 5319 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/3d7f67581a474d359248a68a46f41cdf?auth=ba84c8f9b62a5cf210b8a5cb73f4acd1">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>34. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2369677">test_copy_and_paste_messages</a>, id: 5317 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/28c1eea387484c18a6b21d76bbde43cd?auth=c346abc68fd8ccfb56646acdad5ad95f">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>35. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2369692">test_open_transaction_on_etherscan_copy_tx_hash</a>, id: 5384 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/0edeee05ad904bdfa0f5977eac985875?auth=6ba693a241842e149506a94691851c06">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>36. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2369687">test_unread_messages_counter_public_chat</a>, id: 5360 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/360430bdbb894bb0b921e99d141f706d?auth=fd450b7c8924417c35b851cb6915fd58">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/402583aa26b64815adbc9a778304ab55?auth=b766dcf55335a4a451bb689eddb0784b">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>37. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2369671">test_send_eth_from_wallet_to_address_incorrect_password</a>, id: 5308 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/ccfe23e57fc149179f52eadf3d28300b?auth=2cb939446f0de4e3fe720f1eb9c65666">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>38. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2369713">test_mobile_data_usage_complex_settings</a>, id: 6228 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/9751ef4ac3d24c4db206f42f98bb50dc?auth=43e269c99b0d18d28eefa2023481f707">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/230fdbb2b1a6409383aeacf6c2e59b7f?auth=9658a0c63b9bbb4c0c4ef9b0a36b1f47">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>39. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2369697">test_start_chat_with_ens_mention_in_one_to_one</a>, id: 5403 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/1ec7d42d9bd343f4b967bc3d2ef04f9d?auth=5ab67ebd602fa30023cb41ac7496c942">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>40. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2369721">test_decline_transactions_in_1_1_chat_push_notification_changing_state</a>, id: 6265 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/c6261f1eecd14281be93149fdfaad6d2?auth=647d0fc7be9ddf116817fc21ffa73449">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/2668a441c7124c32b69c51935a0aada4?auth=85187f996e5b48c279f21467c4a49444">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>41. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2369719">test_send_eth_in_1_1_chat_transaction_push</a>, id: 6253 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/a589c6e3c6d2407cb254989b86cbbe80?auth=c87ad67437d078689c6d755e95ff2942">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/5fc8264a65a64e85b698b1c137b60b9a?auth=98100f7a2ace717486f2ba3ee3807402">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>42. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2369706">test_install_pack_and_send_sticker</a>, id: 5782 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/173568738ffa431e89fb6c6ecbe962b3?auth=f4acc14c589d622b8e34bec85dc26663">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/5c07625375134573ac12ff0daea3350a?auth=6c2ee07274d8114aac9727ad681416eb">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>43. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2369686">test_switch_users_special_char_password_and_add_new_account_logcat</a>, id: 5356 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/94fa8dfdf0cd4354b9f92415f04c99ad?auth=629db37e380813e3b5ebbc4a65ab296f">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>44. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2369682">test_manage_assets</a>, id: 5341 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/da5568f702994744874e67315b39ef36?auth=c708d825b1f5ca4261325b0f42b35d8e">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>45. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2369733">test_browser_managing_bookmarks</a>, id: 6633 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/c6824dab56b643e5a50e8da57b763bc0?auth=ee13e5eefe70f8eda9ffb1f2c5aa547b">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>46. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2369670">test_open_chat_by_pasting_chat_key_check_invalid_chat_key_cases</a>, id: 5304 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/465f3cc28fc24327b9d79e5e944da27c?auth=e403d617c76fefc4c0d6311ca907ed90">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>47. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2369707">test_can_use_purchased_stickers_on_recovered_account</a>, id: 5783 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/d0a55affae474cf481c5878824e0ac4e?auth=55d772b8ae1d3ec945270fb50d15018d">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>48. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2369703">test_dapps_permissions</a>, id: 5738 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/6d23491223ae4b708143ec86ee8833c6?auth=a3c04a6e3d046579a1157c4b813a6112">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>49. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2369695">test_account_recovery_with_uppercase_whitespaces_seed_phrase_special_char_passw_logcat</a>, id: 5394 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/38ed2ea573194ead8d5b7c6706253c07?auth=bfc3e3708bea243af2d38f13d5eb4083">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>50. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2369723">test_send_eth_to_ens_in_chat</a>, id: 6279 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/c01c5365b7a04ddcae8cf2f548b1c944?auth=8122b5e85670ffb04951023c0a5897ef">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/dc3d39333262490ea92361588221bc43?auth=38f1dd2a620770dc5c101c0096f81936">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>51. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2369679">test_share_copy_contact_code_and_wallet_address</a>, id: 5323 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/5345eaf9447741e1b2800dcb17c7340a?auth=7ee5d13de6624e4760f5277e6725acae">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>52. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2369681">test_wallet_set_up</a>, id: 5335 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/dfe5d57b17af46088e0c4a7bdee01b0a?auth=bc5d38ca6092311d7c0b850d2c4ccac6">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>53. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2369709">test_send_transaction_with_custom_token</a>, id: 6208 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/107160d8fb8243a0abdc3bc99187023b?auth=b96d69b978762a3af2b7eeecd1cfc4b3">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>54. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2369690">test_home_view</a>, id: 5379 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/fee1c65a7da74263a4bd66414636068e?auth=84953f6f55036753c43f2d2d13b64fd0">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>55. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2369699">test_logcat_backup_recovery_phrase</a>, id: 5419 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/af6ed538fe32478e90e0ec01b689a6e0?auth=696d69da0ac736601351a04c7d605ef6">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>56. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2369720">test_request_and_receive_stt_in_1_1_chat_offline</a>, id: 6263 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/976a6a63716642a4bc005b3357997712?auth=628da3fd44a08f1a6c6d6aa474e28b1a">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/92bcda47a3a24163bb76e2b59dd606fd?auth=fc23c55931c3759f46742bcfca16c94f">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>57. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2369684">test_collectible_from_wallet</a>, id: 5346 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/6eccb0130e1e42f59857c354ae4365f5?auth=903135356f610c8f4c324bd8ac0783f3">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>58. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2369680">test_add_and_remove_mention_contact_with_nickname_from_public_chat</a>, id: 5332 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/25b51bde04e54b649c944fc164e4e493?auth=9d16197f26867f21e35c6753af76105e">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/606c8a65a24840e9901b2beeb2a0e9de?auth=53e774dc55d1420f5d95be59da26218f">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>59. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2369717">test_keycard_request_stt_from_daap</a>, id: 6249 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/b456ab3a0c3242178e7712cfbe9156a3?auth=7b975bbbe311804fbc7c318c80f84370">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>60. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2369715">test_restore_account_from_mnemonic_to_keycard</a>, id: 6240 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/adb74e5a53ed473bbcd749c20e86bda3?auth=b6b505be060b47028126194f2fa3cc45">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>61. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2369688">test_pass_phrase_validation</a>, id: 5363 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/938a2556d8ec4691808ca2c105c92faf?auth=cd3fc44750fbbecd1e6692152a95aa45">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>62. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2369691">test_user_can_see_collectibles_on_rinkeby_after_account_recovering</a>, id: 5381 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/5684f0d9037f4fa7b4fbcb2e4fd4a926?auth=211e370b1a973755b8ba4dc7e7110aab">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>63. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2369727">test_keycard_send_eth_in_1_1_chat</a>, id: 6293 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/feaa50352c4243318a055d61420ee03b?auth=7bffc7525b04ce3e92ef7090994d3327">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/88a491e610cf45e0940b78e0b603cce0?auth=0469de95dff0c6ef5eb62966fd9e90ab">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>64. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2369689">test_send_and_open_links_with_previews</a>, id: 5373 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/b84343f218d241ff819a73cee4bceb19?auth=af9a62a0c9300a3d275c09aa65de4d18">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/26a212fe607f40a2ae14639e9d9f59d3?auth=de130ee78d3ab38a12ee129fd8e02004">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>65. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2369730">test_reactions_to_message_in_chats</a>, id: 6315 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/cb93fe26308a462596c0809155b50ce3?auth=98e7e74087b01b4ff1935f01bc0b3fbc">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/14469231569a4b36bb63ee29fcebdfa9?auth=8e497df0730f0dc73dc0a36b3dc9b2d5">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>66. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2369714">test_fetching_balance_after_offline</a>, id: 6237 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/ce135f4076954467a16b3321b99b841f?auth=0935dff006d52ed995a5d6f3b5ab7e46">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>67. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2369724">test_push_notification_1_1_chat_no_pn_activity_center</a>, id: 6283 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/22b46fc392a04c8b9ddcba3edf3b86c4?auth=cd315de99dcfbcf5e6ecb78580f972a1">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/6a3f4962754d4a3b8d394f6dcc1a57ad?auth=2bb0bac5cf95279831a873c01c78aa94">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>68. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2369674">test_public_chat_messaging_emojis_timestamps</a>, id: 5313 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/457dbe2e5ac145e489fa89965b7d8725?auth=5ce9bc522fd2d37a5494d1923afd6160">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/3b8973b3ca40473a9bdf4da92ff90774?auth=1af99d46defb597f57b7241c0b383a8e">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details>

jakubgs commented:
@status-im/status-testers should I worry about this crop failure?

This PR upgrades a bunch of dependencies as seen in PR description, so weird issues MIGHT happen.

status-im-auto commented:
## 100% of end-end tests have passed
```
Total executed tests: 1
Failed tests: 0
Passed tests: 1
```
<h3>Passed tests (1)</h3><details><summary>Click to expand</summary><br/><br/><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/2372850">test_set_profile_picture</a>, id: 6646 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/e0052d3226014d6ab4a8a2f72953b7f5?auth=7294c6d5ccae1bdfbed77b8ebda5db68">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/6540af2ad6994d389dec777c5645d440?auth=c9f1707b44925fc5294537446a682f15">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details>

qoqobolo commented:
@jakubgs
>should I worry about this crop failure?

Nope, rerun the test and it's passed now.
PR is smoke-tested and can be merged.
